_studio/mfx_lib/shared/include/mfx_frames.h

```diff
@@ -77,31 +77,28 @@
     class clExtTasks1
     {
     public:
-        clExtTasks1()
+        clExtTasks1():
+            m_mGuard()
```

Comment:
emmm. why? this is useless and should be done automatically by compiler.

Reply:
agree